tests/test_collection_get.py

```
@@ -569,6 +872,28 @@ def test_oceanbase_collection_get(self):
             assert "ids" in results
             print(f"   Found {len(results['ids'])} items matching filter")
             
+            # Test 13: Get with scalar $in operator
+            print(f"✅ Testing get with scalar $in operator")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)